.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+'source:web': 'web'
+'source:documentation': 'docs'
+'source:agent':
+  - 'golang/pkg/crane'
+  - 'golang/pkg/dagent'
+  - 'golang/cmd/crane'
+  - 'golang/cmd/dagent'
+'source:ci': '.github/workflows/'
+'source:build':
+  - '**/Makefile'
+  - 'web/kratos'
```

Comment:
We should add a `source:kratos` for that, IMHO. @m8vago your opinion?

Reply:
Done!